### ISSUE TITLE
Missing package.xml?

### ISSUE THREAD

**Issue by zacwitte** [introlab/introlab-ros-pkg#2]
If I put the `x264_image_transport` directory in my catkin workspace it's not found when running catkin_make. I assume it needs `package.xml`.

**Comment by moriarty** [comment]
Yes- we forked this project earlier this week for manyears_ros and the packages are for rosbuild not catkin.

We quickly made some changes to get the many ears package working with catkin, Ubuntu 14.04 and Indigo... But have tasked another student with cleaning up and submitting a pull request.

Are you copying `x264_image_transport` out of this repository and into your catkin workspace? 

Is it worth catkinizing the whole package and making introlab-ros-package a meta-pkg? So that the repository can be clones directly into a catkin workspace?

The packages themselves seem to be very different. If they're no longer actively maintained they could also be broken out into their own repositories. ManyEars is combining audio inputs for determining the source direction and image transport I assume is as the name suggests: users of one won't necessarily need the other.

**Comment by francoisferland** [comment]
About ManyEars, there's already a catkinized and x264-less version of the repository:

https://github.com/introlab/irl_audio

**Comment by moriarty** [comment]
@francoisferland - perfect! 

I missed that when I browsed over your repos.

**Comment by zacwitte** [comment]
I agree they should all be separate repo's. I copied just x264_image_transport into my catkin workspace. I submitted a pull request (#3) that include the catkin-ized files and a few code tweaks to work with newer libraries and ros. 

x264_image_transport is something that the ROS community could really use and it would be great if one of the original authors would make it into a nice package and add some documentation about it on the ROS wiki.

**Comment by siddharthayedida** [comment]
Hello Zacwitte
I tried your catkinized branch . I have a lot warnings as shown below (result of catkin_make). For the same reason, it is not generating ros nodes (binaries). Any thoughts on how to fix this ?
![image](https://user-images.githubusercontent.com/48033887/132607039-5476f9cb-3f38-4c87-a9eb-b5eb42c4765a.png)
